data/talkactions/scripts/create_item.lua

```diff
@@ -28,11 +35,7 @@ function onSay(player, words, param)
 			count = math.max(0, count)
 		end
 	else
-		if not itemType:isFluidContainer() then
-			count = 1
-		else
-			count = 0
-		end
+		count = itemType:isFluidContainer() and 0 or 1
```

Comment:
I meant a function would be useful for the entire `count` definition, not just this line :P e.g.

```
local function getCount(count)
	if count ~= nil then
		if itemType:isStackable() then
			return math.min(10000, math.max(1, count))
		end
		if not itemType:isFluidContainer() then
			return math.min(100, math.max(1, count))
		end
		return math.max(0, count)
	end
	return itemType:isFluidContainer() and 0 or 1
end

(...)

local result = player:addItem(itemType:getId(), getCount(tonumber(split[2])))
```